packages/mui-material/src/locale/index.ts

```diff
@@ -3331,8 +3331,8 @@ export const trTR: Localization = {
           return 'Önceki sayfaya git';
         },
         labelRowsPerPage: 'Sayfa başına satır:',
-        // labelDisplayedRows: ({ from, to, count }) =>
-        //   `${from}–${to} tanesinden ${count !== -1 ? count : `more than ${to}`}`,
+        labelDisplayedRows: ({ from, to, count }) =>
+          `${from}-${to} / ${count !== -1 ? count : `${to}'den fazla'`}`,
```

Comment:
Why did you add `/` for `of` word?

And also remove single quotes around `den fazla`.  
```suggestion
          `${from}-${to} / ${count !== -1 ? count : `den fazla ${to}`}`,
```

Reply:
There is no simple and meaningful way to change the 'of' in Turkish. Slash is global, simple and there is also other languages using slash in the 'index.ts' file.

Secondly, 'more than 2' is translated to Turkish as "2'den fazla", but second single quote is a mistake. I'll fix it.